### ISSUE TITLE
INSANE FRAMELAGG

### ISSUE THREAD

**Issue by caiusp** [HearthSim/Hearthstone-Deck-Tracker#2912]
hi, im having such a hard time with this tracker! im using it from almost 2 years and never had this lag!
in the last 2/3 months went crazy!
i just bought a new gpu(gtx 1060 6gb ram) and didnt change anything.
im running this software on W7 ultimate x64 with i7 and 8gb ram + ssd 250gb , so it shouldnt be a pc performance issue..
can you fix it? 

windows aero is ON.

thanks for your help and keep the good work ! (y)

**Comment by riQQ** [comment]
Does this only occur if you use Hearthstone with HDT or also without it?

**Comment by judge2020** [comment]
See if disabling hardware acceleration does anything, if not set it back to the way it was.

**Comment by caiusp** [comment]
hello guys!
@riQQ i have the same problem without HDT. the problem is the config.file created from HTD , but if i delete it, the tracker wont work :/
@judge2020 where should i disable hardware acceleration??

**Comment by azeier** [comment]
Try adding the Hearthstone directory (and possibly `%LocalAppData%\Blizzard\Hearthstone\Logs`) as an exception in your anti virus software. I started having some similar issues with windows defender after a windows update some time ago.

**Comment by caiusp** [comment]
@azeier thanks for your reply! i already did your step cause i googled it some  days ago for the same issue.
i added this exception to windows defender(altought its disabled) + added every battle.net folder to it.
i dont use an AV, just an anti malware and i did the same.
the problem is this "config.file" , i read it could be an over reading/writing problem but its just an idea..
could it be related to my HDT database?

**Comment by azeier** [comment]
If it's the same with and without HDT running, just by the log.config existing, then this is not a problem with HDT. Check if there's any processes with high CPU usage in the scenario where the file exists and HDT is not running.

**Comment by azeier** [comment]
Or any with high disk usage, actually.

**Comment by caiusp** [comment]
@azeier i'll run some tests in the next hours, thanks for your help!
what do you mean with high disk usage? i have a samsung ssd , less than 2 years old.

**Comment by riQQ** [comment]
You can check the disk usage of process with the Windows `Resource Monitor` (run with `Win + R`, type `resmon` and confirm with Enter). There's a `Disk` tab that displays disk usage statistics of processes.

**Comment by caiusp** [comment]
sorry i had some thinkgs to do IRL and cannot test sooner.
@azeier  i didnt found any suspicious process activity, neither with my W10 laptop.
@riQQ tell me what you need to know [http://imgur.com/a/hrkD3](url)

**Comment by caiusp** [comment]
if we know that the log.config is the problem, what can we do ??

**Comment by riQQ** [comment]
@caiusp was this screenshot taken when you were experiencing lag? It doesn't look like your disk io is maxed out.

**Comment by caiusp** [comment]
@riQQ yep, the screenshot was taken while playing HS w/ HDT. i doubt my disk is maxed out.
Here there s another screenshot taken by Samsung Magician(native FW)  [http://imgur.com/a/xAQaT](url)

**Comment by riQQ** [comment]
You didn't really answer my question? Was it lagging?
Did the lag start after you switched your GPU?

**Comment by caiusp** [comment]
@riQQ i always have lagg and frames dropped: the average is 28-33fps playing with high settings( should be 60fps stable )

**Comment by riQQ** [comment]
It looks like it's the same issue as https://github.com/HearthSim/Hearthstone-Deck-Tracker/issues/1097. I don't have a solution for this.

**Comment by riQQ** [comment]
Did you try deleting `log.config`?

**Comment by caiusp** [comment]
i already checked that old post before posting here :/
yep, i deleted the log.config but the tracker re-create it because it was needed

**Comment by riQQ** [comment]
Yeah it's necessary for HDT working. But your `log.config` could have enabled additional loggers and logging options that are not required for HDT.

**Comment by caiusp** [comment]
@riQQ i think this could def be the problem!
do you want me to update it so you can check it??

**Comment by riQQ** [comment]
If you have deleted it once and started the recent version of HDT, it won't have any unnecessary options set.

**Comment by caiusp** [comment]
@riQQ the old post #1097  has been closed by @azeier  saying that should be fixed with  v0.11.0 
but actually its not..

**Comment by riQQ** [comment]
If your Hearthstone issues are caused by the `log.config` with and without HDT, there's nothing left that can be changed in HDT to fix your problem.

**Comment by s0urc3d3v3l0pm3nt** [comment]
I had the same problem.  It was not a problem with windows defender or logging as far as I can tell.  The fix took me a while to find but it seems to be to remove the deck tracker fully including any data in the appdata directory and do the same for hearthstone.  Then delete log.config (Don't know if this is necessary but can't hurt) and remove and reinstall hearthstone and the deck tracker and your problems should be fixed!

**Comment by caiusp** [comment]
thank you @s0urc3d3v3l0pm3nt  for your reply.
i will def try what you suggested ! :)
